api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -395,6 +396,33 @@ public AuthenticationResponse enrollAuthenticator(ProceedContext proceedContext,
         }
     }
 
+    /**
+     * Verify Authenticator with the supplied authenticator options.
+     *
+     * @param proceedContext the ProceedContext
+     * @return the Authentication response
+     */
+    public AuthenticationResponse verifyAuthenticator(ProceedContext proceedContext) {
+        try {
+            // build answer password authenticator challenge request
+            AnswerChallengeRequest challengeAuthenticatorRequest = AnswerChallengeRequestBuilder.builder()
+                    .withStateHandle(proceedContext.getStateHandle())
+                    .build();
+            String pollHref = proceedContext.getPollHref() != null
+                    ? proceedContext.getPollHref()
+                    : new URL(new URL(proceedContext.getHref()), "/").toString() + "idp/idx/challenge/poll";
```

Comment:
The remediation should return the href to use here. I don't think we want it hard coded.